clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/arrays/ArraysMetadata.scala

```diff
@@ -166,6 +167,19 @@ case class ArraysMetadata(
       notes = appendNote(deletionNote)
     )
 
+  override def withWorkspaceName(name: String): ArraysMetadata = {
+    this.copy(
+      workspaceName = Some(name)
+    )
+  }
+
+  def withMovedIdats(grnIdatPath: URI, redIdatPath: URI): ArraysMetadata = {
```

Comment:
Hrm, do we need this? At the only spot where it's used, the caller knows that it's working with `ArraysMetadata`, so it could just call `copy` directly.

Reply:
Good point, I'll fiddle with that too.